packages/gatsby/src/utils/websocket-manager.ts

```diff
@@ -172,72 +103,6 @@ export class WebsocketManager {
         })
       })
 
-      const getDataForPath = async (path: string): Promise<void> => {
-        const page = findPageByPath(store.getState(), path)
-        if (!page) {
-          socket.send({
-            type: `pageQueryResult`,
-            why: `getDataForPath-notfound`,
-            payload: {
-              id: path,
-              result: undefined,
-            },
-          })
-          return
-        }
-        path = page.path
-
-        let pageData = this.pageResults.get(path)
-        if (!pageData) {
-          try {
-            pageData = await getPageData(path)
-
-            this.pageResults.set(path, pageData)
-          } catch (err) {
-            console.log(err.message)
-            return
-          }
-        }
-
-        const staticQueryHashes = pageData.result?.staticQueryHashes ?? []
-        await Promise.all(
-          staticQueryHashes.map(async queryId => {
-            let staticQueryResult = this.staticQueryResults.get(queryId)
-
-            if (!staticQueryResult) {
-              staticQueryResult = await getStaticQueryData(queryId)
-              this.staticQueryResults.set(queryId, staticQueryResult)
-            }
-
-            socket.send({
-              type: `staticQueryResult`,
-              payload: staticQueryResult,
-            })
-          })
-        )
-
-        socket.send({
-          type: `pageQueryResult`,
-          why: `getDataForPath`,
-          payload: pageData,
-        })
-
-        if (this.clients.size > 0) {
-          telemetry.trackCli(
-            `WEBSOCKET_PAGE_DATA_UPDATE`,
-            {
-              siteMeasurements: {
-                clientsCount: this.clients.size,
-                paths: hashPaths(Array.from(this.activePaths)),
-              },
-            },
-            { debounce: true }
-          )
```

Comment:
This need some thought - I didn't read add it anywhere, but also - this wasn't actually "Page data update" - this was response to `getPageData` request via websocket. If we would need somewhat 1:1 replacement - we could put this in `page-data` express handler